src/content/docs/get-started/index.mdx

```diff
@@ -28,32 +28,25 @@ Welcome to your Commerce Storefront journey. Our goal is to make the journey fun
 
 The following diagram shows the steps you'll take to create and configure your starter storefront:
 
 <Diagram caption="Steps to create and configure your starter storefront.">
-  ![Steps to create and configure your starter storefront.](images/SimpleCreateStorefrontSteps.svg)
-</Diagram>
-
-The full details are slightly more involved. Let's break it down.
-
-<Diagram caption="Details to create and configure your Commerce storefront.">
-  ![Details to create and configure your Commerce storefront.](images/DetailedStorefrontSteps.svg)
+  ![Steps to create and configure your starter storefront.](images/CreateStorefrontSteps.svg)
 </Diagram>
 
 <Callouts>
 
-1. **Use the boilerplate** template to create a storefront repository.
-1. **Add the Code Sync app** to your newly created repo. This app automatically redeploys your storefront when you push changes to the main branch. It also provides the Edge Delivery system (Helix Admin) with access to your repo so it can coordinate code changes with content changes.
-1. **Create a folder** on Google Drive or SharePoint for your content and _Share_ it with the `helix@adobe.com` user. This gives the Edge Delivery system (Helix Admin) with read/write access to your folder's content.
-1. **Add sample content** from the boilerplate to your new folder on Google Drive or SharePoint.
-1. **Connect your repo to your content** using the `mountpoint` URL in the **`fstab.yaml`** in your GitHub repo.
-1. **Connect your site to your data** using the values in the **`config.json`** in your GitHub repo.
-1. **Install and configure Sidekick** so it can preview, publish, and edit content on your storefront.
-1. **Set up your local development environment** and install the dependencies. Start the development server and explore your new storefront.
+1. **Create Site Repo** by generating a storefront repository from the [boilerplate template](https://github.com/hlxsites/aem-boilerplate-commerce).
+1. **Link Repo to Content** using the `mountpoint` URL in the **`fstab.yaml`** in your GitHub repo. You can use [default-fstab.yaml](https://github.com/hlxsites/aem-boilerplate-commerce/blob/main/default-fstab.yaml) as a reference.
+1. **Add Code Sync App** to your newly created repo. This app automatically redeploys your storefront when you push changes to the main branch. It also provides the Edge Delivery system (Helix Admin) with access to your repo so it can coordinate code changes with content changes.
+1. **Add Content** in https://da.live/ using the [import tool](https://da.live/app/hlxsites/aem-boilerplate-commerce/tools/site-creator/site-creator) or the [site importer](https://da.live/apps/import), if you already have a site in SharePoint or Google Drive.
```

Comment:
It seems like a good idea to clarify what https://da.live is since it's the first introduction of the URL.

Changed link to point to the Site Import documentation instead of the interface, which is not easy to figure out.

```suggestion
1. **Add Content** to the Document Author environment hosted at `https://da.live/` using the [import tool](https://da.live/app/hlxsites/aem-boilerplate-commerce/tools/site-creator/site-creator). If you already have a site in Sharepoint or Google Driver, you can  or the [site importer](https://da.live/docs/operations/import), if you already have a site in SharePoint or Google Drive.
```

Reply:
Done.